addons/mail/static/tests/tours/discuss_sidebar_in_public_page_tour.js

```diff
@@ -36,5 +36,14 @@ registry.category("web_tour.tours").add("sidebar_in_public_page_tour", {
         {
             trigger: ".o-mail-DiscussSidebarChannel:contains(Channel 2).o-active",
         },
+        {
+            content: "Open channel actions (should hover sidebar channel item to be visible)",
+            trigger: "[title='Channel Actions']:not(:visible)",
+            run: "click",
+        },
+        {
+            trigger: ".o-dropdown-item:contains('Invite People')",
+            run: "click",
+        },
```

Comment:
You can `hover && click` to click on `Channel Actions` and making it visible from hover.
This is not visible in CSS but it could have been implemented with `t-if` rather than `d-none` on non-hover. the `hover && click` works regardless of implementation details.